src/arrow2/src/io/parquet/read/deserialize/nested_utils.rs

```diff
@@ -455,8 +456,7 @@ pub(super) fn extend<'a, D: NestedDecoder<'a>>(
 /// * `decoded`          - The state of our decoded values.
 /// * `decoder`          - The decoder for the leaf-level type.
 /// * `additional`       - The number of top-level rows to read for the current chunk. This is the
-///                        min of `chunk size - number of rows existing in the current chunk` and
-///                        `rows_remaining`.
+///   min of `chunk size - number of rows existing in the current chunk` and `rows_remaining`.
```

Comment:
The vertical spacing beforehand seemed good. Why remove it?

Reply:
This is the result of a clippy lint. In any case the actual docstring when you hover over it is the same.